templates/index.html

```diff
@@ -1,18 +1,65 @@
+<div class="jumbotron" style=background-color:#0a0a0a;">
+  <h1 style="height: 100 px; background:linear-gradient(to bottom, #33ccff 0%, #ff99cc 100%)" class="display-3 text-center">&#129412; LINDSEY'S TO-DO APP &#129412;</h1>
+   
+  <Hr color="white"></Hr>
 
-  <div class="row">
+      <p style="height: 60 px; background:linear-gradient(to right, #ff99cc 0%, #33ccff 100%)" class="display-5 text-center"> &#128151; Lindsey Raven &#128151;</p>
+      
+<div class="row">
     <div class="col-md-12">
-      <h2>Items</h2>
-      <ul class="list-group mb-4">
-        <!-- List the todo items here -->
-      </ul>
-    </div>
-  </div>
+      <h2 style="color:#ffffff">Lindsey's Items</h2>
+        <ul class="list-group mb-4">
+          <br/>
+          
+          <!-- List / Delete form -->
+            {% for todo in todos %}
+              <li class="list-group-item text-info">
+                {{todo.title}} <br/>
+              <body style="height: 1000 px; background:linear-gradient(to top, #ff99cc 0%, #33ccff 100%);"></body>
+                <span class="badge badge-secondary" style="background-color: plum;">{{todo.id}}</span>
+                <span class="badge badge-warning" style="background-color:#d39ef7">{{todo.status}}</span>
+                <div class="btn-group float-right" role="group" aria-label="item actions">           
+
+                <!-- status update started -->
+
+              <form action="/started-todo" method="POST"> 
+                <input type="hidden" name="todo_id" value='{{ todo.id }}'>
+                <input type="submit" class="btn btn-secondary btn-sm" style="color:#e4acca !important" value="Started"> 
+              </form>   
+              <br/>
+
+               <!-- status update completed -->
+
+              <form action="/complete-todo" method="POST">
+                <input type="hidden" name="todo_id" value='{{ todo.id }}'>
+                <input type="submit" class="btn btn-dark btn-sm" style="color:#e4acca !important" value="Completed"> 
+              </form>   
+              
+                <!-- status update delete -->
+
+                <form action="/delete-todo" method="POST">
+                  <input type="hidden" name="todo_id" value='{{ todo.id }}'>
+                  <input type="submit" class="btn btn-dark btn-sm" style="background-color:#111313" :#e4acca !important" value="Delete"> 
+                </form>
+               
+            {% endfor %}
+        </div>
+        
+        <!-- ADD fORM -->
+
+        <div class="col-md-12 text-center">
+          <form action="/add-todo" method="POST">
+            <br/>
+
+            <span style='font-size:25px;'>&#9997;</span>
+              <label style="text-decoration-color: #fcdeee" for="name">Enter your to-do name:</label>
+              <input class="text-info" type="text" name="name" id="name">
+            <input class="text-info" type="submit" class="btn btn-dark btn-md" style="color:#b41369 !important" value="Submit"
```

Comment:
This `<input>` element is missing a `>` (this currently breaks the submit button)

Reply:
> This `<input>` element is missing a `>` (this currently breaks the submit button)

cheers! must had deleted it by mistake and not realises. > has now been added back in. I need to learn to expand my VS code that way I can see everything all at once.